### PR TITLE
Add 'bluesky-queueserver-api' package

### PR DESCRIPTION
**Pull request by dmgav**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/bluesky-queueserver-api/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - httpx
+    - python >=3.7
+
+test:
+  imports:
+    - bluesky_queueserver_api
+  commands:
+    - pip check
+  requires:
+    - pip
```
Comment (mrakitin) [inline recipes/bluesky-queueserver-api/meta.yaml:32]: As the `pip check` fails for Windows, let's exclude this check for that platform.
```suggestion
  commands:      # [not win]
    - pip check  # [not win]
  requires:      # [not win]
    - pip        # [not win]
```

**Review by mrakitin — COMMENTED** [review]
@conda-forge/staged-recipes, I agree to be listed as a maintainer.

File: recipes/bluesky-queueserver-api/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{% set name = "bluesky-queueserver-api" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bluesky-queueserver-api-{{ version }}.tar.gz
```
Comment (mrakitin) [inline recipes/bluesky-queueserver-api/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

**Comment by dmgav** [comment]
@conda-forge/staged-recipes, the PR is ready for review

**Pull request merged**